modin/logging/class_logger.py

```diff
@@ -31,10 +31,12 @@ class ClassLogger:
     This mixin must go first in class bases declaration to have the desired effect.
     """
 
+    _modin_logging_layer = "PANDAS-API"
```

Comment:
If you mean declare the string as a constant outside of the class (and re-use it in this line to set the variable) then yes, sure, it will work. But why?..

Unless we want to re-use the same constant in the `@enable_logging`...

Reply:
Sorry, was responding to the comment here: https://github.com/modin-project/modin/pull/4583#discussion_r902463682

I thought you were saying this didn't work as we wanted it to, my mistake. Is this ok to merge?

Reply:
Unless you want me to extract this constant (which I don't see much value in, TBH), it should be good to go.